src/modules/70-pipeline/components/PipelineSteps/AdvancedSteps/FailureStrategyPanel/utils.ts

```diff
@@ -85,3 +86,5 @@ export function handleChangeInStrategies({
 export function getTabIntent(i: number, selectedStrategyNum: number): Intent {
   return i === selectedStrategyNum ? Intent.PRIMARY : Intent.NONE
 }
+
+export const EXCLUDE_STEP_INSIDE_STEP_GROUP_FAILURE_STRAT_LIST = [Strategy.RetryStepGroup]
```

Comment:
```suggestion
export const EXCLUDE_STEP_INSIDE_STEP_GROUP_FAILURE_STRATEGY_LIST = [Strategy.RetryStepGroup]
```
We can keep the full name, as the name itself is long now